curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-algorithmic-thinking-by-building-a-number-sorter/6407c6d3f19c4e0a7ba320bb.md

```diff
@@ -0,0 +1,320 @@
+---
+id: 6407c6d3f19c4e0a7ba320bb
+title: Step 12
+challengeType: 0
+dashedName: step-12
+---
+
+# --description--
+
+In your `sortInputArray()` function, call your `updateUI()` function and pass `inputValues` as the argument.
```

Comment:
In your `sortInputArray()` function, call your `updateUI()` function and pass in `inputValues` as the argument.

Reply:
All of the documentation I've seen uses `pass an argument`, not `pass in an argument`.

Reply:
Ok.

Apart from those, I don't have any other thing for now. I have pulled in the PR locally so I can see how the tests work. I'll get back to you on that.